js/bitget.js

```diff
@@ -1502,417 +1365,187 @@ module.exports = class bitget extends Exchange {
+        //       size: 0.001,
+        //       orderId: '881640729145409536',
+        //       clientOid: '881640729204129792',
+        //       filledQty: 0.001,
+        //       fee: 0,
+        //       price: null,
+        //       priceAvg: 38429.5,
+        //       state: 'filled',
+        //       side: 'open_long',
+        //       timeInForce: 'normal',
+        //       totalProfits: 0,
+        //       posSide: 'long',
+        //       marginCoin: 'USDT',
+        //       filledAmount: 38.4295,
+        //       orderType: 'market',
+        //       cTime: '1645925450611',
+        //       uTime: '1645925450746'
         //     }
         //
-        let id = this.safeString (order, 'order_id');
-        id = this.safeString2 (order, 'id', 'data', id);
-        const timestamp = this.safeInteger2 (order, 'created_at', 'createTime');
-        let type = this.safeString (order, 'type');
-        const side = this.parseOrderSide (type);
-        type = this.parseOrderType (type);
-        // if ((side !== 'buy') && (side !== 'sell')) {
-        //     side = this.parseOrderSide (type);
-        // }
-        // if ((type !== 'limit') && (type !== 'market')) {
-        //     if ('pnl' in order) {
-        //         type = 'future';
-        //     } else {
-        //         type = 'swap';
-        //     }
-        // }
         const marketId = this.safeString (order, 'symbol');
-        market = this.safeMarket (marketId, market);
-        const amount = this.safeString2 (order, 'amount', 'size');
-        const filled = this.safeString2 (order, 'filled_amount', 'filled_qty');
-        const cost = this.safeString (order, 'filled_cash_amount');
+        market = this.safeMarket (marketId);
+        const symbol = market['symbol'];
+        const id = this.safeString2 (order, 'orderId');
         const price = this.safeString (order, 'price');
-        const average = this.safeString (order, 'price_avg');
-        const status = this.parseOrderStatus (this.safeString2 (order, 'state', 'status'));
-        const feeCost = this.safeNumber2 (order, 'filled_fees', 'fee');
-        let fee = undefined;
-        if (feeCost !== undefined) {
-            const feeCurrency = undefined;
-            fee = {
-                'cost': feeCost,
-                'currency': feeCurrency,
-            };
+        const amount = this.safeString2 (order, 'quantity', 'size');
+        const filled = this.safeString2 (order, 'fillQuantity', 'filledQty');
+        const cost = this.safeString2 (order, 'fillTotalAmount', 'filledAmount');
+        const average = this.safeString (order, 'fillPrice');
+        const type = this.safeString (order, 'orderType');
+        const timestamp = this.safeInteger (order, 'cTime');
+        let side = this.safeString2 (order, 'side', 'posSide');
+        if ((side === 'open_long') || (side === 'close_short')) {
+            side = 'buy';
+        } else {
+            side = 'sell';
```

Comment:
@frosty00 `spot` order will have `side`: `sell` and `buy`, is it intended to have this `if` statement turn everything into `sell`